WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -669,6 +685,42 @@ extension ProductsViewController: SyncingCoordinatorDelegate {
 
         ServiceLocator.stores.dispatch(action)
     }
+
+    /// Update local Products Settings (eg. sort order or filters stored in Products settings)
+    ///
+    func updateLocalProductSettings(sort: ProductsSortOrder? = nil,
+                                    stockStatusFilter: ProductStockStatus? = nil,
+                                    productStatusFilter: ProductStatus? = nil,
+                                    productTypeFilter: ProductType? = nil) {
+        let action = AppSettingsAction.upsertProductsSettings(siteID: siteID,
+                                                              sort: sort?.rawValue,
+                                                              stockStatusFilter: stockStatusFilter,
+                                                              productStatusFilter: productStatusFilter,
+                                                              productTypeFilter: productTypeFilter) { (error) in
+        }
+        ServiceLocator.stores.dispatch(action)
+    }
+
+    /// Fetch local Products Settings (eg.  sort order or filters stored in Products settings)
+    ///
+    func syncLocalProductsSettings(onCompletion: @escaping (Result<StoredProductSettings, Error>) -> Void) {
+        let action = AppSettingsAction.loadProductsSettings(siteID: siteID) { [weak self] (result) in
+            switch result {
+            case .success(let settings):
+                if let sort = settings.sort {
+                    self?.sortOrder = ProductsSortOrder(rawValue: sort) ?? .nameAscending
```

Comment:
⚠️ I suggest that we make the default `. nameAscending` as a part of `ProductsSortOrder` since `.nameAscending` is also used above to define the “default”. Maybe something like:

```swift
public enum ProductsSortOrder: String {
    static let default = .nameAscending
}
```

And then this can be:

```suggestion
                    self?.sortOrder = ProductsSortOrder(rawValue: sort) ?? .default
```